### ISSUE TITLE
Filtering graph

### ISSUE THREAD

**Issue by smarr** [eregon/virtual-machines-research-graph#6]
I'd like to be able to focus the graph on specific aspects.

- based on concepts: see what's most related only
- based on edges: filter specific kinds of edges (the implementation knows a few types of edges (concept edges, extends and use edges) but it is generally open for other kinds

Posting this mostly as note for myself, but also because it may relate to #5.

**Comment by eregon** [comment]
Another way to approach this would be to have the graph be "explore-able" and the ability to e.g. focus on a single node and expand as needed (a bit like in IGV).

**Comment by smarr** [comment]
Yeah, the question is, do we really want to implement this?

Just did some searching. visjs might be interesting:
- https://visjs.github.io/vis-network/examples/network/data/dynamicFiltering.html
- https://visjs.github.io/vis-network/examples/network/data/datasets.html

more examples: https://visjs.github.io/vis-network/examples/

**Comment by eregon** [comment]
> the question is, do we really want to implement this?

Yes of course we should find an existing library, I'm not interested to reimplement visualization :D
